File: Code/BeforeLegends/Assets/Scripts/Test stuff/RotateTile.js
#pragma strict

var player : GameObject;
var pos : Vec2i;

function Update () {
	if(Input.GetKeyDown(KeyCode.G)) {
		pos = player.GetComponent.<MapObjectCarrier>().pos;
		var worlddata : WorldMapData = WorldMapData.getInstance();
		//worlddata.tiles[pos.x, pos.y].gameObject;
	}
}

File: Code/BeforeLegends/Assets/Scripts/Test stuff/RotateTile.cs
using UnityEngine;
using System.Collections;

public class RotateTile : MonoBehaviour {

    GameObject player;
    Vec2int pos;

    void Update () {
	    if(Input.GetKeyDown(KeyCode.G)) {
		    pos = player.GetComponent<MapObjectCarrier>().pos;
            WorldMapData worlddata = WorldMapData.instance;
		    //worlddata.tiles[pos.x, pos.y].gameObject;
	    }
    }
}
